src/examples/src/config.tsx

```diff
@@ -502,6 +503,12 @@ export const config = {
 					filename: 'ListBox',
 					module: ListBox,
 					title: 'List Box'
+				},
+				{
+					description: 'This example shows the menu rendering 100,000 options.',
```

Comment:
It's not really rendering 100,000 options, perhaps we can better describe it.

Reply:
sure